validator/src/commands/wait_for_restart_window/mod.rs

```diff
@@ -68,19 +94,15 @@ pub(crate) fn command(default_args: &DefaultArgs) -> App<'_, '_> {
 }
 
 pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
-    let min_idle_time = value_t_or_exit!(matches, "min_idle_time", usize);
-    let identity = pubkey_of(matches, "identity");
-    let max_delinquent_stake = value_t_or_exit!(matches, "max_delinquent_stake", u8);
-    let skip_new_snapshot_check = matches.is_present("skip_new_snapshot_check");
-    let skip_health_check = matches.is_present("skip_health_check");
+    let wait_for_restart_window_arg = WaitForRestartWindowArgs::from_clap_arg_match(matches);
```

Comment:
nit: Let's rename this to match too. `wait_for_restart_window_arg` ==> `wait_for_restart_window_args`

Reply:
52fa0f29ae